src/app/page.tsx

```diff
@@ -1,46 +1,55 @@
 "use client";
 
 import { useEffect, useState } from "react";
-
 import { Button } from "@/components/ui/button";
 import Link from "next/link";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Share2 } from "lucide-react"; 
 
 export default function Home() {
   const [signatureCount, setSignatureCount] = useState<number | null>(null);
 
   useEffect(() => {
-    // EventSource를 사용하여 SSE 연결
     const eventSource = new EventSource("signature/size");
 
-    // 데이터를 수신하면 상태 업데이트
     eventSource.onmessage = (event) => {
       const { size } = JSON.parse(event.data);
       setSignatureCount(size);
     };
 
-    // 오류 처리
     eventSource.onerror = (error) => {
       console.error("EventSource error:", error);
       eventSource.close();
     };
 
-    // 컴포넌트 언마운트 시 연결 닫기
     return () => {
       eventSource.close();
     };
   }, []);
 
+  const handleShare = () => {
+    if (typeof navigator !== "undefined" && navigator.clipboard?.writeText) {
```

Comment:
navigator.clipboard?.writeText 로 하지 말고 
```suggestion
    if (typeof navigator !== "undefined" &&  navigator.clipboard && navigator.clipboard.writeText) {
```